opensearch.tf

```diff
@@ -6,7 +6,8 @@ data "aws_iam_policy_document" "opensearch_for_github" {
       "es:Describe*",
       "es:List*",
       "es:GetCompatibleVersions",
-      "es:GetUpgradeHistory"
+      "es:GetUpgradeHistory",
+      "es:StartDomainMaintenance",
```

Comment:
could the trailing comma be removed from the end of `"es:StartDomainMaintenance",`